tables/articles.csv

```diff
@@ -370,7 +370,7 @@ Sabin21,10.1101/2021.10.02.21264415,NULL,"Sabin, A",2021,2021-12-07,NULL
 Sepulcri21,10.1093/ofid/ofab217,NULL,"Sepulcri, C",2021,2021-12-07,NULL
 Cele21c,10.1101/2021.12.08.21267417,NULL,"Cele, S",2021,2021-12-07,2021-12-14
 Wilhelm21b,10.1101/2021.12.07.21267432,NULL,"Wilhelm, A",2021,2021-12-08,NULL
-Cao21,10.1101/2021.12.07.470392,NULL,"Cao, YR",2021,2021-12-10,NULL

```

Comment:
Please update the date_updated column, please don't change the date_added column. Also, the date is 2022-01-18.